### ISSUE TITLE
provide a mean to get the extradata

### ISSUE THREAD

**Issue by lu-zero** [ultravideo/kvazaar#108]
a `kvazaar_extradata` basically would need this

``` c
    // Video Parameter Set (VPS)
    kvz_nal_write(stream, KVZ_NAL_VPS_NUT, 0, 1);
    encoder_state_write_bitstream_vid_parameter_set(state);

    // Sequence Parameter Set (SPS)
    kvz_nal_write(stream, KVZ_NAL_SPS_NUT, 0, 1);
    encoder_state_write_bitstream_seq_parameter_set(state);

    // Picture Parameter Set (PPS)
    kvz_nal_write(stream, KVZ_NAL_PPS_NUT, 0, 1);
    encoder_state_write_bitstream_pic_parameter_set(state);
```

provided in a separate bitstream and then chunks.

(it is useful to be able to put the encoded data in mkv and mp4 without going through additional bitstream filters).

**Comment by aryla** [comment]
Implemented in 979ab6c.

**Comment by lu-zero** [comment]
The version hadn't been bumped though.
